frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/EmbeddingDataPicker.tsx

```diff
@@ -105,11 +105,8 @@ export function EmbeddingDataPicker({
       selectedDatabaseId={databaseId}
       selectedTableId={pickerInfo?.tableId}
       selectedCollectionId={card?.collection_id}
-      databaseQuery={{ saved: true }}
```

Comment:
we always call `{ saved: true }` inside the `DataSelector` component, so there's no point passing the same value.